kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/SchemaManager.java

```diff
@@ -503,9 +507,12 @@ TableInfo constructTableInfo(TableId table, com.google.cloud.bigquery.Schema big
             "Cannot create/update BigQuery table for record with no value schema. "
             + "If delete mode is enabled, it may be necessary to enable schema unionization to handle this case.");
       }
-      return com.google.cloud.bigquery.Schema.of();
     }
-    com.google.cloud.bigquery.Schema valueSchema = schemaConverter.convertSchema(kafkaValueSchema);
+
+    com.google.cloud.bigquery.Schema valueSchema = com.google.cloud.bigquery.Schema.of();;
```

Comment:
If we got a null schema with schema unionization turned off, I don't think we would even get here. Having deletes be predicated on schema unionization seems unintuitive. What is the rationale for this limitation?